src/app/containers/PageViewAnalytics/Canonical/getPageViewBeaconUrl.test.js

```diff
@@ -0,0 +1,105 @@
+import * as genericLabelHelpers from '../labelHelpers';
+import * as articleLabelHelpers from '../labelHelpers/article';
+
+const mockAndSet = ({ name, source }, response) => {
+  source[name] = jest.fn(); // eslint-disable-line no-param-reassign
+  source[name].mockImplementation(() => response);
+};
+
+const getAtiUrl = obj => require('./getPageViewBeaconUrl').default(obj); // eslint-disable-line global-require
+
+const splitUrl = url =>
```

Comment:
Personally I'd prefer a helper const which was named similar to `splitOnChar` rather than `split()` being instantly followed by `join()`. 
```
const splitOnChar = (url, char) => url.split(char).join(',');
```

Then this line can become easier to read like so: 

```suggestion
const splitUrl = url => url.splitOnChar('?').splitOnChar('&').split(',');
```

Reply:
I like the principle but `url.splitOnChar('?').splitOnChar('&').split(',')` wont work as `splitOnChar` isnt a protorype method of Array

Reply:
Good point, how about ...

```
const splitUrl = url => {
  let commaSeperatedUrl = splitOnChar(url, '?');
  commaSeperatedUrl = splitOnChar(commaSeperatedUrl, '&');
  return commaSeperatedUrl.split(',');
}
```

Which is now so long that it's not worth the change 😞